Index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="setting.css">
+    <title>landing-page</title>
+</head>
+<body>
+    <header class="header">
+        <img src="./product-landing-page-logo.png" alt="" id="header-img">
+        <nav class="nav-bar">
+<ul>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">Features</li></a>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">How It Works</li></a>
+    <li><a  href="https://product-landing-page.freecodecamp.rocks/#features">Pricing</li></a>
+</ul>
+     </nav>
+    </header>
+
+    <section>
+    <div class="form">
+        <h2>Handcrafted, home made Masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+            <input
+            type="email"
+            placeholder="Enter your email addres"
+            name="email"
+            id="email"
+            readonly
+            />
+            <input id="submit" type="submit" value="Get Started" />          
+        </form>
+    </div> 
```

Comment:
your work is good to modify and align the margin and the padding  property of the layout smart 
and align the centre and   validation all input filed

Reply:
> your work is good to modify and align the margin and the padding property of the layout smart and align the centre and validation all input filed

- [ ] You should make sure your comments are clear and easy to understand, so the owner of the code understand them clearly. You could say for example :+1: 
Your work look good ,it could be better to modify and align the margin, the padding property of the layout so it looks smart. You could also align all the input field in the centre :+1: 
What do you this about this comment above :+1: